.circleci/config.yml

```diff
@@ -22,8 +22,8 @@ jobs:
       - run:
           name: install python dependencies
           command: |
-            curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | POETRY_VERSION=1.1.8 python
-            . $HOME/.poetry/env
+            pip install --upgrade pip
```

Comment:
Have you tried replacing get-poetry.py with https://github.com/python-poetry/install.python-poetry.org? e.g. `curl -sSL https://install.python-poetry.org | POETRY_VERSION=1.1.15 python3 -`? I'm not sure if this is better than using pip install but it looks it looks more in line with the way it was done before.